examples/desk_security_guard.py

```diff
@@ -48,8 +48,8 @@
 class TwitterStreamToAppCommunication:
     '''Class for messaging to/from SecurityGuardStreamListener
 
-    Tweepy doesn't support asyncio, so we have to run the SecurityGuardStreamListener
-    stream in its own thread. We limit the communication between the rest of the
+    Tweepy doesn't support asyncio, so it has to run the SecurityGuardStreamListener
```

Comment:
i don't think it is correct in this context (it's not Tweepy running this, it's this program running it), so if "we" is bad grammar then perhaps "so this program has to"?